spec/renuo/bin-check/bin_check_spec.rb

```diff
@@ -19,9 +19,32 @@
       end
     end
 
-    expect(BinCheck.new_script_hash)
-      .to eq(rubocop: { files: 'lib/**/*.rb', command: 'rubocop', reversed_exit: false },
-             t0d0_finder: { files: 'lib/**/*.*', command: 'blubb-blubb', reversed_exit: true })
+    expect(BinCheck.configs.first.configs)
+      .to eq(name: 'rubocop', files: 'lib/**/*.rb', command: 'rubocop', reversed_exit: false)
+    expect(BinCheck.configs[1].configs)
+      .to eq(name: 't0d0_finder', files: 'lib/**/*.*', command: 'blubb-blubb', reversed_exit: true)
```

Comment:
just a detail, but maybe we could replace the `t0d0_finder` with another test that would not bother our `bin/check` :)